### PR TITLE
Internal: Elements - Make elements dropping as one user action [ED-6979]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: assets/dev/js/editor/commands/drop.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export class Drop extends $e.modules.CommandBase {
+	apply( args = {} ) {
+		elementor.getPreviewContainer().view.createElementFromModel( args );
```
Comment (matipojo) [inline assets/dev/js/editor/commands/drop.js:3]: return value

File: assets/dev/js/editor/component.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Documents from './components/documents/component';
+import BrowserImport from './components/browser-import/manager';
+
+import * as commands from './commands/';
+
+export default class Component extends $e.modules.ComponentBase {
+	getNamespace() {
+		return 'editor';
```
Comment (matipojo) [inline assets/dev/js/editor/component.js:8]: Do we have a `preview` component?

```suggestion
		return 'preview';
```

File: assets/dev/js/editor/component.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	}
+
+	registerAPI() {
+		super.registerAPI();
+
+		elementor.browserImport = new BrowserImport();
+		elementor.documents = $e.components.register( new Documents() );
```
Comment (matipojo) [inline assets/dev/js/editor/component.js:15]: The `documents` is not related to the preview.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/views/base-container.js [hunk 4]
```diff
@@ -161,14 +161,13 @@ module.exports = Marionette.CompositeView.extend( {
...
-					.filter( ( [ key ] ) => [ 'elType', 'widgetType', 'custom' ].includes( key ) )
-			),
-			options
+		const args = Object.fromEntries(
+			Object.entries( elementor.channels.panelElements.request( 'element:selected' )?.model.attributes )
+				// The `custom` property is responsible for storing global-widgets related data.
+				.filter( ( [ key ] ) => [ 'elType', 'widgetType', 'custom' ].includes( key ) )
 		);
+
+		$e.run( 'editor/drop', args );
```
Comment (iNewLegend) [inline assets/dev/js/editor/views/base-container.js:170]: add options

**Pull request merged**